apps/web/scripts/installExternalCourse.sh

```diff
@@ -4,3 +4,8 @@ wget "https://github.com/$1/archive/refs/heads/main.zip" -O temp.zip
 unzip temp.zip "*/course/*" -d ../../courses/$2
 mv ../../courses/$2/*/course/* ../../courses/$2
 rm temp.zip
+
+wget "https://codeberg.org/Lamdarer/LibreLingo-DE-from-EN/archive/main.zip" -O temp.zip
```

Comment:
This is weird, now this will download this zip file every time any course is imported.

how about instead using the whole URL as a parameter? So that it can be called like this:


```bash
yarn installCourse "https://github.com/kantord/LibreLingo-ES-from-EN/archive/refs/heads/main.zip" spanish-from-english
yarn installCourse https://codeberg.org/Lamdarer/LibreLingo-DE-from-EN/archive/main.zip german-from-english
```

Reply:
I dont really know, do as you think its best

Reply:
I think that will be good for now, supplying the whole link. That way it's not dependent on the provider (github vs codeberg)

do you need help making the changes?

Reply:
Maybe, I changed apps/web/scripts/installExternalCourses.sh like this as you wrote, do I need to change anything else?(You can also just do the changes)
```
yarn installCourse "https://github.com/kantord/LibreLingo-ES-from-EN/archive/refs/heads/main.zip" spanish-from-english
yarn installCourse https://codeberg.org/Lamdarer/LibreLingo-DE-from-EN/archive/main.zip german-from-english
```

Reply:
sure, here's the MR with the changes: https://github.com/kantord/LibreLingo/pull/1659/files